WhatsApp Shortcuts.user.js

```diff
@@ -53,9 +58,87 @@
             }
 
             return currentReply.nextSibling;
-        }
+        },
+
+        getConversation(){
+            return document.querySelectorAll('#pane-side .X7YrQ div._2UaNq');
+         }
     }
 
+    function addEventListenerToConversation(){
+        const conversationsElements = WhatsApp.getConversation();
+        WhatsApp.conversation.totalConversationElements= conversationsElements.length;
+
+        conversationsElements.forEach((conversationElement, index) => {
+            conversationElement.addEventListener('click',() => {
+                WhatsApp.conversation.currentConversationIndex = index;                    
+            });
+        });
+    }
+
+    function bindChangeOfConversation(){
+
+        const event = new MouseEvent('onMouseDown', {
+            'view': window,
+            'bubbles': true,
+            'cancelable': true
+        });
+
+        Mousetrap.bind(['down'], function() {
+        try{
+
+
+            const currentIndex = WhatsApp.conversation.currentConversationIndex;
+            const totalIndex = WhatsApp.conversation.totalConversationElements;
+
+            const nextIndex = (totalIndex + currentIndex - 1) %totalIndex;                
+
+            const nextElement = WhatsApp.getConversation()[nextIndex];
+            if(!nextElement) return;                
+
+            const reactHandlerKey=Object.keys(nextElement).filter(function(item){
+                return item.indexOf('__reactEventHandlers')>=0
+             });
+            const reactHandler=nextElement[reactHandlerKey[0]];
```

Comment:
Would you mind to extract this to a function, then we can call it in `up` and `down`